From b84c4ba97cecf7304e99cafdd8a9c5866ce27050 Mon Sep 17 00:00:00 2001
From: Hartmut Goebel <h.goebel@crazy-compilers.com>
Date: Tue, 21 Jan 2020 23:33:50 +0100
Subject: [PATCH] Fix missing link libraries.

See <https://phabricator.kde.org/D26819>

These are only actually missing if the libraries reside in different
prefixes, as it is the case in Guix or Nix.
---
 resources/ews/test/CMakeLists.txt | 1 +
 resources/facebook/CMakeLists.txt | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/resources/ews/test/CMakeLists.txt b/resources/ews/test/CMakeLists.txt
index b20eddcb8..6355eb994 100644
--- a/resources/ews/test/CMakeLists.txt
+++ b/resources/ews/test/CMakeLists.txt
@@ -35,6 +35,7 @@ qt5_add_resources(isolatestestcommon_RSRCS isolatedtestcommon.qrc)
 add_library(isolatedtestcommon STATIC ${isolatestestcommon_SRCS})
 target_link_libraries(isolatedtestcommon
   KF5::AkonadiCore
+  KF5::AkonadiMime
   Qt5::Core
   Qt5::Network
   Qt5::Test
diff --git a/resources/facebook/CMakeLists.txt b/resources/facebook/CMakeLists.txt
index bdd5eeaa7..27a9c83c1 100644
--- a/resources/facebook/CMakeLists.txt
+++ b/resources/facebook/CMakeLists.txt
@@ -21,7 +21,8 @@ add_library(facebookresourcelib STATIC ${fbresource_SRCS})
 
 target_link_libraries(facebookresourcelib
     KF5::KIOWidgets
+    KF5::AkonadiCore
     KF5::IconThemes
     KF5::I18n
     KF5::ConfigGui
     KF5::CalendarCore
-- 
2.21.1

